src/NzbDrone.Core.Test/ParserTests/LanguageParserFixture.cs

```diff
@@ -165,6 +165,7 @@ public void should_parse_language_bulgarian(string postTitle)
 
         [TestCase("Movie.Title.1994.Dublado.1080p.XviD-LOL")]
         [TestCase("Movie.Title.2.2019.1080p.Bluray.Dublado.WWW.TPF.GRATIS")]
+        [TestCase("Movie.Title.3.2021.1080p.Bluray.Brazilian.WWW.TPF.COM.BR")]
```

Comment:
Can you add one with BRAZiLiAN.PORTUGUESE in as well please so that all bases are covered :) (as per @bakerboy448  message in the conversation)

Reply:
Sorry, i've been busy. Added a couple more tests based on what i see on private trackers.

Actually BRAZiLiAN.PORTUGUESE breaks the test since it expects PTBR but it matches PT and PTBR. I'm not sure on how to handle it, any suggestions?

I didn't wanted to parse Portuguese like this: 
`lowerTitle.Contains("portuguese") && !lowerTitle.Contains("brazilian.portuguese")`

But i can't think of anything else right now.

Reply:
@Qstick thoughts on how to handle?